docs/basic-features/image-optimization.md

```diff
@@ -121,7 +121,8 @@ The following Image Optimization cloud providers are included:
 - [Imgix](https://www.imgix.com): `loader: 'imgix'`
 - [Cloudinary](https://cloudinary.com): `loader: 'cloudinary'`
 - [Akamai](https://www.akamai.com): `loader: 'akamai'`
-- dangerously-unoptimized: uses image directly from `src` prop without optimization (only needed with `next export` or when the built-in `_next/image` optimizer can not be leveraged) `loader: 'dangerously-unoptimized'`
+- Custom: `loader: 'custom'` use a custom cloud provider by implementing the [`loader`](/docs/api-reference/next/image.md#loader) prop on the `next/image` component
+- Dangerously Unoptimized: `loader: 'dangerously-unoptimized'` uses image directly from `src` prop without optimization
```

Comment:
Maybe this should be called `dangerously-default-unoptimized` to convey it's a default, you can still add the `loader` prop. This part of the docs should mention 

When adding `dangerously-default-unoptimized` we can warn in development when `dangerously-default-unoptimized` is set without a `loader` provided to `next/image` 👍

Reply:
I'm starting to think we should combine `dangerously-unoptimized` and `custom` values here.

Let's just do `custom` and anyone that wants to do something dangerous can set `loader={(src) => src}` manually.

In fact, we can probably detect if their custom `loader()` function doesn't return the `w` param and warn.

Reply:
I like that!